### PR TITLE
Long form can

### PR DESCRIPTION
**Pull request by sglyon**
reshapes CAN data to be long form

### CODE REVIEW

File: schemas/can_tables.sql [hunk 1]
```diff
@@ -9,208 +9,138 @@ INSERT INTO meta.actnow_intervention_types (id, name) VALUES
...
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_deaths is E'The proejcted number of cumulative COVID-19 deaths';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_infected is E'The projected, cumulative number of persons infected with COVID-19.';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_positive_tests is E'The projected, cumulative number of people who have tested positive for COVID-19.';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_negative_tests is E'The projected, cumulative number of people who have tested negative for COVID-19.';
-
-
-DROP TABLE IF EXISTS data.actnow_state_timeseries;
-CREATE TABLE data.actnow_state_timeseries (
-    "dt" DATE,
+INSERT INTO meta.actnow_actual_variables (id, name) VALUES
```
Comment (cc7768) [inline schemas/can_tables.sql:36]: Would it be particularly crazy to combine `meta.actnow_timeseries_variables` and `meta.actnow_actual_variables`? Lots of similar columns and might make the variable coding cleaner.

File: src/cmdc/datasets/covidactnow/data.py [hunk 2]
```diff
@@ -65,23 +66,26 @@ def get(self):
...
         temp_cols = list(df)
         temp_cols[intervention_ix] = "it.id as intervention_id"
+        temp_cols[variable_ix] = "vt.id as variable_id"
 
         out = f"""
         INSERT INTO data.{table_name} ({", ".join(final_cols)})
         SELECT {", ".join(temp_cols)}
         from {temp_name} tt
         LEFT JOIN meta.actnow_intervention_types it on it.name = tt.intervention
+        LEFT JOIN meta.{self.table_name + "_variables"} vt on vt.name = tt.variable
```
Comment (cc7768) [inline src/cmdc/datasets/covidactnow/data.py:88]: I see -- The column names probably come directly from CAN data which you can just use...

Disregard both comments. This all looks good.

**Review by cc7768 — APPROVED** [review]
Looks good to me.

File: schemas/can_tables.sql [hunk 3]
```diff
@@ -9,208 +9,138 @@ INSERT INTO meta.actnow_intervention_types (id, name) VALUES
...
+(9, 'hospital_beds_typical_usage_rate'),
+(10, 'icu_beds_capacity'),
+(11, 'icu_beds_current_usage_covid'),
+(12, 'icu_beds_current_usage_total'),
+(13, 'icu_beds_total_capacity'),
+(14, 'icu_beds_typical_usage_rate');
+
+DROP TABLE IF EXISTS data.actnow_timeseries;
+CREATE TABLE data.actnow_timeseries (
+    "date" DATE,
```
Comment (cc7768) [inline schemas/can_tables.sql:54]: We should change all `date` columns in the database to `dt` for consistency.

**Review by cc7768 — CHANGES_REQUESTED** [review]
Just noticed that the date columns are called `date` in the database -- It makes sense to go ahead and change those to `dt` to reflect changes made in the rest of the tables.

File: schemas/can_tables.sql (hunk 1 above)
Comment (sglyon) [reply schemas/can_tables.sql:36]: they are totally different

one is input the other is outputs

**Pull request merged**